management/azure_mgmt_compute/spec/virtual_machines_spec.rb

```diff
@@ -113,11 +113,11 @@
     ni = network_client.make_request(:get, ni_path, options)
     expect(ni).not_to be_nil
     # user NetworkInterface mapper to deserialize object
-    ni_instance  = NetworkInterface.new.from_json(ni)
+    ni_instance  = network_client.deserialize(NetworkInterface.mapper(), ni)
     # retrieve first IP address for first IP configuration
     ip_address_path = ni_instance.ip_configurations[0].public_ipaddress.id
     ip_address = network_client.make_request(:get, ip_address_path, options)
-    ip_address_instance = PublicIPAddress.new.from_json(ip_address)
+    ip_address_instance = network_client.deserialize(PublicIPAddress.mapper(), ip_address)
```

Comment:
just curious what has caused this change?

Reply:
I was working with David on a different task. It was dropped eventually. This was a remnant from that task